node/src/container_chain_spawner.rs

```diff
@@ -674,9 +783,14 @@ fn handle_update_assignment_state_change(
     }
 }
 
-/// Select `SyncMode` to use for a container chain.
-/// We want to use warp sync unless the db still exists, or the block number is 0 (because of a warp sync bug in that case).
-/// The reason is that warp sync doesn't work if a database already exists, it falls back to full sync instead.
+/// Select [SyncMode] to use for a container chain.
+/// We want to use warp sync unless the container chain is still at genesis block (because of a warp sync bug in that case).
```

Comment:
```suggestion
/// We want to use warp sync unless the db still exists, or the container chain is
/// still at genesis block (because of a warp sync bug in that case).
```